src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -9186,6 +9186,7 @@ public enum CompareOptions
         IgnoreSymbols = 4,
         IgnoreKanaType = 8,
         IgnoreWidth = 16,
+        NumericOrdering = 32,
         OrdinalIgnoreCase = 268435456,
         StringSort = 536870912,
         Ordinal = 1073741824,
```

Comment:
nit: should we change this to use hex numbers for the sake of the readability?

Reply:
This is how the GetAPI tool generates it and the guidance I've heard is to make as few diffs from that tool as possible.

Reply:
Yes, but still can make diffs :-)

Reply:
The ref assembly isn’t really designed for readability, it’s designed to be correct and automatically generated

Deviations and manual diffs just cause later downstream pain and hinder the ability to rerun the tool, as people have to fight against it 

The better option would be to submit a bug or better a patch such that the output produced by the tool for flags enabled enums is “better” such as using hex or logical shifts to represent the bits instead (`1 << 0, 1 << 1, 1 << 2`, etc)

Reply:
I've created an issue: https://github.com/dotnet/sdk/issues/44999. GenAPI seems to be in a kind of limbo state where there is a new Roslyn based version we want to switch to (tracked by https://github.com/dotnet/sdk/issues/31088) and we don't want to maintain the current CCI-based one (https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.GenAPI/README.md).